app/controllers/case_studies_controller.rb

```diff
@@ -77,6 +77,21 @@ def search
   def update
     @case_study = CaseStudy.find(params[:id])
 
+    if Rails.env.production?
```

Comment:
Same note here as below, refactoring this as a method on the model would allow code reuse.